wom/src/main/resources/reference.conf

```diff
@@ -4,3 +4,10 @@ yaml {
   # The maximum nested depth for nodes (sequences + mappings) that will be parsed in a YAML
   max-depth = 1000
 }
+
+womParse {
+  # Assume there is WDL code that has a nested scatter. Should we distinguish between
+  # the inner and outer loops? The default is yes. Different code is created for the inner
+  # loop vs. the outer loop.
+  inner-outer-scatter = true
```

Comment:
This is a slightly awkward name. Could it become something like `convert-nested-scatter-to-subworkflow`?